### ISSUE TITLE
Meetings with meetingId yy6vwndol0ras8lpskeyjjsryzhgznwwnlok4cmm can't be started

### ISSUE THREAD

**Issue by Ithanil** [blindsidenetworks/scalelite#1120]
**Describe the bug**
Get yourself some popcorn.

A CREATE call with the abovementioned `meetingId=yy6vwndol0ras8lpskeyjjsryzhgznwwnlok4cmm` and *without* a specified `voiceBridge` (or external voice bridge disabled) will lead to a meeting with `voiceBridge=99999` (which shouldn't be an issue in itself?) and an internal meeting ID of e.g. `-1730718410132`, which is *only* the timestamp part. This is a problem, because e.g. further requests like a JOIN call towards that meeting will fail:
![image](https://github.com/user-attachments/assets/4b1af77a-1ccd-438f-aed7-29b197b2a36f)

***Steps to reproduce the behavior:***

1. Prepare a Scalelite with at least one BBB Server added (can be 2.7 or 3.0, doesn't matter).
2. Prepare an API-Mate window with the secret of SL and use as `meetingId` (and `name`, but doesn't matter) the value `yy6vwndol0ras8lpskeyjjsryzhgznwwnlok4cmm`. Make sure the field `voiceBridge` is empty.
3. Execute the create call, observe the fields `voiceBridge` and `internalMeetingID` in the response.
4. Execute the join call, observe the error message from the screenshot above.

**Expected behavior**

The meeting should be created with sane `voiceBridge` and `internalMeetingID` values. It should be possible to join the meeting without issues.

**Additional context**

I think the problem is caused here: https://github.com/blindsidenetworks/scalelite/blob/3880dcda5aac78d7a4cbb19ee0d872f81de0a199/app/models/meeting.rb#L203
Although I do not yet fully understand what exactly is going wrong, removing https://github.com/blindsidenetworks/scalelite/blob/3880dcda5aac78d7a4cbb19ee0d872f81de0a199/app/models/meeting.rb#L209 and creating the prng without pre-determined seed like `prng = Random.new()` resolves the issue.

**Comment by Ithanil** [comment]
> Interesting - Can you please open an issue against BigBlueButton (or link one if you have already opened it) for the problem where meetings fail to start if the voice bridge number is all "9"s?
> 
> The purpose for seeding the RNG based on the meeting ID is to make the voice bridge number (relatively) stable for meetings - i.e. multiple instances of the same meeting will tend to have the same voice bridge number. This is convenient for when the BigBlueButton servers are configured for dial-in support.

Thanks for your comment! 

I am going to open an issue later in the evening, but have to move right now (it's already late here).
I strongly suspect the only affected voice bridge is the number set for `testVoiceBridge` in `/usr/share/bbb-web/WEB-INF/classes/bigbluebutton.properties` (or corresponding override file).

**Comment by kepstin** [comment]
> We are using dial-in support, but basically understood the voice bridge numbers as completely (pseudo-)random / unpredictable so far.

This is mostly dependent on the external integration you are using. In order to get the same conference number, it has to re-use the same meeting ID for a series of meetings. If it's creating a new meeting ID each time, then it can't make use of this feature. (Re-using the same meeting ID also makes retrieving recordings more efficient, so it's recommended when possible)

> I strongly suspect the only affected voice bridge is the number set for `testVoiceBridge` in `/usr/share/bbb-web/WEB-INF/classes/bigbluebutton.properties` (or corresponding override file).

It would make sense to add a configuration option to scalelite for this, then - provide the test voice bridge number as a configuration option, and have the voice bridge number generation automatically generate a new number in case of conflicts. It could even be set to match BigBlueButton's default by default.

**Comment by Ithanil** [comment]
> It would make sense to add a configuration option to scalelite for this, then - provide the test voice bridge number as a configuration option, and have the voice bridge number generation automatically generate a new number in case of conflicts. It could even be set to match BigBlueButton's default by default.

Yeah, that's basically what I recommended as a workaround in my issue above. Just that the parameter has actually nothing to do with a `maxVoiceBridge` and should be named properly + made configurable.

Nevertheless, I feel like this is still an issue with BBB. The API reference explicitly states
> Voice conference number for the FreeSWITCH voice conference associated with this meeting. This must be a 5-digit number in the range 10000 to 99999.
and does not mention a special `testVoiceBridge` number as an exception. Thus, the BBB server does not behave as expected, even if the configuration is left at default.
I will open an issue for BBB and let you guys decide what to do with this. ;-)

**Comment by Ithanil** [comment]
Fixed by https://github.com/blindsidenetworks/scalelite/pull/1128
